crates/forge/tests/it/invariant.rs

```diff
@@ -240,7 +240,7 @@ async fn test_invariant_shrink() {
         CounterExample::Single(_) => panic!("CounterExample should be a sequence."),
         // `fuzz_seed` at 119 makes this sequence shrinkable from 4 to 2.
         CounterExample::Sequence(sequence) => {
-            assert!(sequence.len() == 2)
+            assert_eq!(sequence.len(), 2);
```

Comment:
cc @brockelmore seems this sequence is not getting shrinked as expected—i'll investigate, but wanted to see if you had thoughts / ideas as you recently implemented a new shrinking algo